### ISSUE TITLE
Interactive apps menu panel should be a landmark

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#704]
The Interactive apps menu panel should be a navigation landmark. Otherwise folks have to tab through a whole lot to get there. 

Or maybe every category be a landmark.  That would help us with the "All Apps page"

For the "My interactive sessions" it would go here:
https://github.com/OSC/ondemand/blob/583a955db1dfb4e27f49c99612b00ba4540a6937/apps/dashboard/app/views/batch_connect/sessions/index.html.erb#L18

If we did every category it would be here. 
https://github.com/OSC/ondemand/blob/583a955db1dfb4e27f49c99612b00ba4540a6937/apps/dashboard/app/views/batch_connect/shared/_app_list.html.erb#L1

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1201737641781652) by [Unito](https://www.unito.io)

**Comment by matt257** [comment]
reviewed,
part of this issue is reviewing and finding current landmarks
